scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -595,7 +594,7 @@
                                         (sort-by (comp :db/id :job #(.getRequest %))))
         :let [request (.getRequest task)
               task-id (:task-id request)
-              job-id (get-in request [:job :db/id])]]
+              job-id [:job/uuid (get-in request [:job :job/uuid])]]]
```

Comment:
I see, you are creating a look up ref instead of using the entity id. I think it is worth re-naming to make that clear.